edb/lib/std/60-baseobject.edgeql

```diff
@@ -44,6 +44,11 @@ CREATE ABSTRACT TYPE std::Object EXTENDING std::BaseObject {
         'Root object type for user-defined types';
 };
 
+CREATE TYPE std::VirtualObject EXTENDING std::BaseObject {
```

Comment:
This needs to be `final`, I think

Reply:
Agreed that that would be good, unfortunately final doesn't work currently: https://github.com/edgedb/edgedb/issues/2607